packages/builder/src/pages/builder/app/[application]/_layout.svelte

```diff
@@ -112,23 +223,100 @@
         <VersionModal />
         <RevertModal />
         <Icon name="Play" hoverable on:click={previewApp} />
-        <DeployModal />
+
+        {#if isPublished}
+          <PopoverMenu
+            bind:this={publishPopover}
+            align="right"
+            disabled={!isPublished}
+            dataCy="publish-popover-menu"
+          >
+            <div slot="control" class="icon app-status-icon">
+              <Icon
```

Comment:
The designs included a normal tooltip even when the app is published, so I think we should add that in too.
![image](https://user-images.githubusercontent.com/9075550/165077075-c19a049e-d673-4065-82ff-e1f418bacfd6.png)

Reply:
Ah yes, my mistake.